src/test/java/hw_jdi/pages/MetalsAndColorsPage.java

```diff
@@ -0,0 +1,184 @@
+package hw_jdi.pages;
+
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.List;
+
+import static org.testng.Assert.assertTrue;
+
+public class MetalsAndColorsPage {
```

Comment:
Take a look at the classwork and entity driving approach. 
You have missed one layer of abstraction in your PO model. (mrof)